amounts.go

```diff
@@ -30,6 +71,15 @@ func ParseDecimalStrict(s string) (*big.Rat, error) {
 	return v, nil
 }
 
+// MustParseStellarAmount is the panicking version of ParseStellarAmount.
+func MustParseStellarAmount(v string) int64 {
+	ret, err := ParseStellarAmount(v)
+	if err != nil {
+		panic(err)
+	}
+	return ret
+}
```

Comment:
prefer not to have panic, needed for something?

Reply:
only used here: https://github.com/keybase/client/pull/13983/files#diff-0b5bcf545339bfd2298a79f1b1e66572R341. pretty easy to rip out considering it's a test.